### PR TITLE
Fix routing through friendly rooms

### PR DESCRIPTION
**Pull request by Somotaw**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Somotaw/screeps/pull/2" target="_blank">ReviewNinja

File: src/prototype_creep_routing.js [hunk 1]
```diff
@@ -23,9 +23,11 @@ Creep.prototype.getRoute = function() {
 
       if (Memory.rooms[roomName] && Memory.rooms[roomName].state == 'Occupied') {
         //         console.log(`Creep.prototype.getRoute: Do not route through occupied rooms ${roomName}`);
-        if (config.allowRoutingThroughFriendRooms && friends.indexOf(Memory.rooms[roomName].player) > -1) {
+        if (config.path.allowRoutingThroughFriendRooms && friends.indexOf(Memory.rooms[roomName].player) > -1) {
```
Comment (Somotaw) [inline src/prototype_creep_routing.js]: !fix test this 
not tested yet !fixed

Comment (Somotaw) [inline src/prototype_creep_routing.js]: !fixed

File: src/prototype_creep_routing.js [hunk 2]
```diff
@@ -23,9 +23,11 @@ Creep.prototype.getRoute = function() {
 
       if (Memory.rooms[roomName] && Memory.rooms[roomName].state == 'Occupied') {
         //         console.log(`Creep.prototype.getRoute: Do not route through occupied rooms ${roomName}`);
-        if (config.allowRoutingThroughFriendRooms && friends.indexOf(Memory.rooms[roomName].player) > -1) {
```
Comment (Somotaw) [inline src/prototype_creep_routing.js]: blah

File: src/prototype_creep_routing.js (hunk 1 above)
Comment (Somotaw) [inline src/prototype_creep_routing.js]: :thumbsup:

**Comment by Somotaw** [comment]
👍

**Comment by Somotaw** [comment]
👎

**Comment by Somotaw** [comment]
👍

**Pull request closed without merging**